site/source/_posts/2018/03/11/Dojo-2-rc1.md

```diff
@@ -185,13 +185,13 @@ Dojo 2 also makes it easy to support other patterns such as those popularized by
 
 ## Documentation and examples
 
-To get started, we have a [series of Dojo tutorials and documentation](https://dojo.io/docs/) ranging from the very basic to more complicated topics such as lazy loading, web animations and more. In addition to these, all Dojo 2 packages have detailed READMEs.
+To get started, we have a [series of Dojo tutorials and documentation](https://dojo.io/docs/fundamentals/accessibility/index.html) ranging from the very basic to more complicated topics such as lazy loading, web animations and more. In addition to these, all Dojo 2 packages have detailed READMEs.
```

Comment:
I don't think this is where I would link first... perhaps to the tutorials list?

Reply:
This is the default URL that click docs on dojo.io navigates to... if you just use `https://dojo.io/docs/` it just goes to a blank page which is odd